### DISCUSSION TITLE
angular or vue

### DISCUSSION THREAD

**Discussion by hjs-wcl** [DouyinFE/semi-design#1390] [General]
有无计划启动angular或vue3

**Comment by pointhalo** [comment]
没有这方面的计划。Semi Design 是一个由抖音内部技术产品孵化而来的开源项目，我们在内部有较多现存活跃项目在使用 Semi ，我们团队自身是 React 的重度用户，我们需要同时服务社区用户与公司内部的用户。如果需要维护其他技术栈的组件库，对于我们来说，并不能保证足够的人力投入。
并且如果我们自身并未在大量在真实项目中使用验证的话，质量其实无法得到保障 ，这并非是一个好的选择。

React 体系的library 依然会是我们工作的重点，至于未来会考虑 WebComponent ，但目前没有明确的推进计划。
